### ISSUE TITLE
memory analysis tab disappear after a while

### ISSUE THREAD

**Issue by garanews** [spender-sandbox/cuckoo-modified#240]
Is it normal that with memory analysis enabled  I obtain corretctly the memory analysis tab
![image](https://cloud.githubusercontent.com/assets/16938405/17330742/4d7a442c-58c8-11e6-849f-083ffd9acac1.png)

and after a while it disapppears?

Below the config

retention.conf

[retention]
enabled = yes
# run at most once every this many hours (unless reporting.conf is modified)
run_every = 6
# The amount of days old a task needs to be before deleting data
# Set a value to no to never delete it
memory = 14
procmemory = 90
pcap = 90
sortedpcap = 14
bsonlogs = 90
dropped = 90
screencaps = 90
reports = 90
mongo = 180
elastic = no
malheur = 180

memory.conf 
[basic]
# Profile to avoid wasting time identifying it
# Note: mem_profile in the machinery configuration will override this
guest_profile = Win7SP1x64
# Delete memory dump after volatility processing.
dostrings = yes
strings_nullterminated_only = no
strings_minchars = 5
delete_memdump = no
# Delete memory dump in the event of a volatility exception
delete_memdump_on_exception = no

# List of available modules
# enabled: enable this module
# filter: use filters to remove benign system data from the logs
# Filters are defined in the mask section at below

# Scans for hidden/injected code and dlls
# http://code.google.com/p/volatility/wiki/CommandReferenceMal23#malfind
[malfind]
enabled = yes
filter = on

**Comment by DigiAngel** [comment]
I see this as well....after a complete analysis the tab is missing.

**Comment by spender-sandbox** [comment]
Yes, you have the retention module enabled and are telling it to delete full memory dumps after 14 days.

-Brad
